src/webgldrawer.js

```diff
@@ -1067,63 +1071,85 @@
             this._clippingContext.restore();
         }
 
+            /**
```

Comment:
This is so minor I feel silly mentioning it, but you've got a stray tab here...

Reply:
Fixed